app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,52 @@
+class AnswersController < ApplicationController
+
+
+  def new
+    @answer = Answer.new
+  end
+
+  def create
+    answer = Answer.new(create_answer_params)
+    answer.user_id = session[:user_id]
```

Comment:
It would be better to use current_user here. You have current_user in application_controller - you may as well use it as it encapsulates what a current user is. It's no beyond the bounds of possibility that that could change as your application grows. If it does, you don't want to have to find and refactor all the places you refer to session[:user_id]